service/src/main/kotlin/no/nav/su/se/bakover/service/revurdering/RevurderingServiceImpl.kt

```diff
@@ -646,10 +647,26 @@ internal class RevurderingServiceImpl(
                         }.left()
                     }
 
+                val leggTilVarselForBeløpsendringUnder10Prosent =
+                    !VurderOmBeløpsendringErStørreEnnEllerLik10ProsentAvGjeldendeUtbetaling(
+                        eksisterendeUtbetalinger = eksisterendeUtbetalinger.flatMap { it.utbetalingslinjer },
+                        nyBeregning = beregnetRevurdering.beregning,
+                    ).resultat
+
                 when (beregnetRevurdering) {
                     is BeregnetRevurdering.IngenEndring -> {
                         revurderingRepo.lagre(beregnetRevurdering)
-                        identifiserFeilOgLagResponse(beregnetRevurdering).right()
+                        when (leggTilVarselForBeløpsendringUnder10Prosent) {
```

Comment:
Er `BeregnetRevurdering.IngenEndring` nå død kode?

Reply:
Det stemmer, vi vil aldri ende opp der for nye revurderinger - inntil vi legger tilbake en sperre på et senere tidspunkt.